hub/collections/dataset/__init__.py

```diff
@@ -132,3 +133,17 @@ def merge(datasets: Iterable[Dataset]) -> Dataset:
             assert tname not in tensors
             tensors[tname] = tvalue
     return Dataset(tensors)
+
+
+def from_tensorflow(input):
```

Comment:
Need to have test_{function_name} added to a `tests/test_dataset.py` or another `tests/test_[python_file_name].py`